app/assets/stylesheets/finder_frontend.scss

```diff
@@ -446,3 +446,9 @@
     }
   }
 }
+
+/* Override class to bleed more into govuk-form-group class */
+.govuk-skip-link:focus {
```

Comment:
We shouldn't be adding glue code like this - can this be built into the component instead?

Reply:
We shouldn't need too add this to component, its just used to add spacing. but I've fixed that